src/services/outliningElementsCollector.ts

```diff
@@ -212,18 +212,32 @@ namespace ts.OutliningElementsCollector {
             return spanForNode(node, /*autoCollapse*/ false, /*useFullStart*/ !isArrayLiteralExpression(node.parent), open);
         }
 
-        function spanForNode(hintSpanNode: Node, autoCollapse = false, useFullStart = true, open: SyntaxKind.OpenBraceToken | SyntaxKind.OpenBracketToken = SyntaxKind.OpenBraceToken): OutliningSpan | undefined {
+        function spanForNode(
+            hintSpanNode: Node,
+            autoCollapse = false,
+            useFullStart = true,
+            open: SyntaxKind.OpenBraceToken | SyntaxKind.OpenBracketToken | SyntaxKind.OpenParenToken = SyntaxKind.OpenBraceToken,

```

Comment:
Why did this change? I'm not seeing any change in callers.

Reply:
Previously this was `open: SyntaxKind.OpenBraceToken | SyntaxKind.OpenBracketToken`, now there's a third possibility.